### PR TITLE
img2pdf: add pt_BR translation

### PR DESCRIPTION
**Pull request by Felipe-noob**
- [ ] The page (if new), does not already exist in the repository.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

I translated the page I have recently submitted. Can I translate several commands and submit all of them at once or should I submit each one individually?

### CODE REVIEW

File: pages.pt_BR/common/pactree.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# img2pdf
+
+> Ferramenta de conversão sem perdas de imagens para PDF.
+> Mais informações: <https://gitlab.mister-muffin.de/josch/img2pdf>.
+
+- Converter múltiplas imagens para um único PDF, cada imagem sendo uma página:
+
+`img2pdf {{caminho/da/imagem1}} {{caminho/da/imagem2.jpg}} --output {{caminho/do/arquivo}}`
```
Comment (pixelcmtd) [inline pages.pt_BR/common/pactree.md:8]:
```suggestion
`img2pdf {{caminho/da/imagem1.jpg}} {{caminho/da/imagem2.jpg}} --output {{caminho/do/arquivo.pdf}}`
```

File: pages.pt_BR/common/pactree.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Converter para PDF apenas o primeiro quadro de uma imagem com múltiplos quadros:
+
+`img2pdf {{caminho/do/arquivo.gif}} --first-frame-only --output {{path/to/file.pdf}}`
```
Comment (pixelcmtd) [inline pages.pt_BR/common/pactree.md:12]:
```suggestion
`img2pdf {{caminho/do/arquivo.gif}} --first-frame-only --output {{caminho/do/arquivo.pdf}}`
```

File: pages.pt_BR/common/pactree.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`img2pdf {{caminho/do/arquivo.jpg}} --auto-orient --pagesize {{A4^T}} --border {{2cm}}:{{5.1cm}} --output {{caminho/do/arquivo.pdf}}`
+
+- Encolher apenas imagens maiores para um retângulo de 10cm por 15cm dentro de uma página de 30x20cm:
+
+`img2pdf {{caminho/do/arquivo.tiff}} --pagesize {{30cm}}x{{20cm}} --imgsize {{10cm}}x{{15cm}} --fit {{shrink}} --output {{caminho/do/arquivo.pdf}}`
+
+- Converter uma imagem para PDF e especificar os metadados do arquivo resultante:
+
+`img2pdf {{caminho/do/arquivo.png}} --title {{título}} --author {{autor}} --creationdate {{1970-01-31}} --keywords {{palavra-chave1 palavra-chave2}} --subject {{assunto}} --output {{caminho/do/arquivo.pdf}}`
```
Comment (pixelcmtd) [inline pages.pt_BR/common/pactree.md:24]:
```suggestion
`img2pdf {{caminho/do/arquivo.png}} --title {{título}} --author {{autor}} --creationdate {{1970-01-31}} --keywords "{{palavra-chave1 palavra-chave2}}" --subject {{assunto}} --output {{caminho/do/arquivo.pdf}}`
```

Comment (Felipe-noob) [reply pages.pt_BR/common/pactree.md:24]: Why the quotation marks? The original version didn't use them (should I change it?). I could use _ instead of -.

**Review by blueskyson — APPROVED** [review]
LGTM.

File: pages.pt_BR/common/pactree.md (hunk 3 above)
Comment (navarroaxel) [reply pages.pt_BR/common/pactree.md:24]: :thinking: How are the keyword treated when are space separated? are there 2 diff keywords?

Comment (pixelcmtd) [reply pages.pt_BR/common/pactree.md:24]: If it works this way, you don't need to change it. But it really confuses me, _how_ it works.

Comment (marchersimon) [reply pages.pt_BR/common/pactree.md:24]: From the source code:
```Python
metaargs.add_argument(
        "--keywords",
        metavar="kw",
        type=str,
        nargs="+",
        help="Sets the keywords metadata value (can be given multiple times)",
)
```
This looks like the example above shouldn't even work and it should be `--keyword {{keyword1}} --keyword {{keyword2}}`.

**Comment by marchersimon** [comment]
> Can I translate several commands and submit all of them at once or should I submit each one individually?

Multiple translations can be submitted in one PR, but when submitting new (unrelated) commands, it should be done in a seperate PR.

File: pages.pt_BR/common/pactree.md (hunk 3 above)
Comment (Felipe-noob) [reply pages.pt_BR/common/pactree.md:24]: I tested this and it seems like `--keyword {{keyword1 keyword2}}` saves the two words as different keywords. `--keyword "{{keyword1 keyword2}}"` will save them to just one keyword according to Okular but two according to `exiftool`. `--keyword {{keyword1}} --keyword {{keyword2}}` doesn't work, only keyword2 will be registered.

Comment (pixelcmtd) [reply pages.pt_BR/common/pactree.md:24]: That is horrible, crappy, confusing and doesn't make any sense at all. But your page is correct, marking this as resolved.

**Review by marchersimon — APPROVED** [review]
Thanks @Felipe-noob

**Review by pixelcmtd — APPROVED** [review]

File: pages.pt_BR/common/pactree.md (hunk 3 above)
Comment (navarroaxel) [reply pages.pt_BR/common/pactree.md:24]:
```suggestion
`img2pdf {{caminho/do/arquivo.png}} --title {{título}} --author {{autor}} --creationdate {{1970-01-31}} --keywords {{palavra_chave1 palavra_chave2}} --subject {{assunto}} --output {{caminho/do/arquivo.pdf}}`
```

File: pages.pt_BR/common/pactree.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
```
Comment (navarroaxel) [inline pages.pt_BR/common/pactree.md:24]: The placeholders should be in `snake_case`:
```suggestion
`img2pdf {{caminho/do/arquivo.png}} --title {{título}} --author {{autor}} --creationdate {{1970-01-31}} --keywords {{palavra_chave1 palavra_chave2}} --subject {{assunto}} --output {{caminho/do/arquivo.pdf}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Comment by navarroaxel** [comment]
Congrats @Felipe-noob, please clean up your fork and avoid send PRs from your `Felipe-noob:main` branch, that branch should be on sync the the `tldr-pages:main`. Now, you could make a `git reset --hard HEAD~100` in your `main` or re-fork to pull commits from `tldr-pages:main` in a clean way.

**Comment by marchersimon** [comment]
@Felipe-noob would you like to help translating our alias page template? You'd just have to put the `pt_BR` translation of
```
# example

> This command is an alias of `example`.

- View documentation for the original command:

`tldr example`
```
to `contributing-guides/translation-templates/alias-pages.md`. Thanks!

**Comment by navarroaxel** [comment]
@marchersimon we already have that translation but Felipe can review it :grimacing:

**Comment by marchersimon** [comment]
Oh, right. We just forgot to mark it as done in #5986

**Comment by Felipe-noob** [comment]
It seems OK to me, I think I would understand the message as an end-user. I'm not sure if my compatriots prefer using "pseudonym", "synonym", "codename", or the foreignism "alias". My colleagues don't agree on a word, so I'd leave it as it is. It's good in my opinion. Over the next weeks, I intend to translate a few pages and submit them.

**Pull request merged**